cvcl/models.py

```diff
@@ -284,3 +284,58 @@ class Meta:
 
     def __str__(self):
         return str(self.id)
+
+
+class InstallPackage(models.Model):
+    name = models.CharField(max_length=30)
+    package_update = models.BooleanField(default=False)
```

Comment:
"This module allows packages to be updated, upgraded or installed during boot. If any packages are to be installed or an upgrade is to be performed then the package cache will be updated first. " If this is being applied to all packages .Why not have a parent package class and a child package class that inherits those properties